_posts/2020-04-07-trying-out-julia.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: Trying out Julia programming language
+author: hnybom
+excerpt: >
+  This post is about the Julia programming language and my journey starting from zero knowledge to, well, some knowledge of the language. So if you, like me, are interested trying out Julia language keep on reading. If you are already familiar with the subject you can safely skip this one.
```

Comment:
are interested **in** trying

Jos nyt oikein kaivellaan, niin ehkä tästä koko excerptistä vois kirjoittaa vähän "myyvemmän" :) Mutta luetaanpa ensin koko blögäytys.

Reply:
Lisäsin vähän "flarea" tohon.